interpreter.js

```diff
@@ -163,14 +163,25 @@ Interpreter.generateAST = function(code, parseOptions) {
 /**
  * Add more code to the interpreter.
  * @param {string|!Object} code Raw JavaScript text or AST.
+ * @param {Object} additionalOptions Parsing options to pass through to acorn.
  */
-Interpreter.prototype.appendCode = function(code) {
+Interpreter.prototype.appendCode = function(code, additionalOptions) {
```

Comment:
can we set a default value for `additionalOptions` here instead of doing it manually on lines 169-171?

```suggestion
Interpreter.prototype.appendCode = function(code, additionalOptions = {}) {
```

Reply:
Unfortunately, that's an ES6 feature.